tests/unit/disco/components/TestFooter.js

```diff
@@ -22,7 +22,7 @@ describe('Footer', () => {
 
     expect(root.find('.Footer-privacy-link')).toHaveProp(
       'href',
-      'https://www.mozilla.org/privacy/websites/',
+      'https://www.mozilla.org/privacy/websites/?utm_source=discovery.addons.mozilla.org&utm_medium=firefox-browser&utm_content=privacy-policy-link',
```

Comment:
Should we be hardcoding this expectation, or should we be using the `makeQueryStringWithUTM` function to generate it?